src/core/tests/visitors/op/group_normalization.cpp

```diff
@@ -14,7 +14,7 @@ using namespace ov;
 using ov::test::NodeBuilder;
 
 TEST(attributes, group_normalization) {
-    NodeBuilder::get_ops().register_factory<ov::op::v10::Unique>();
+    NodeBuilder::opset().insert<ov::op::v12::GroupNormalization>();
```

Comment:
Didn't it change the test result? The changed itself is fine - I'm just curious.

Reply:
Without change this was test error as builder has not this operator in opset and cand build, visit attributes for tested operator. 

Previous implementation  was using `FactoryRegistry` which on first use load one of opset table and operator could be in factory map. New implementation is using `OpSet` class which "test_opset" which has empty factory map.
This change detect test issues not only for this operator.